README.md

```diff
@@ -197,8 +197,25 @@ cd timelessis
 pip install -r requirements.txt
 ```
 
-### Database Setup
+### Setup PostgreSQL for Debian / Ubuntu
+Debian / Ubuntu repositories include PostgreSQL by default. To install PostgreSQL, use the apt-get (or other apt-driving) command:
+```
+apt-get update
```

Comment:
I think we should add `sudo` here, WDYT?

Reply:
I just tried these commands under root user in docker container. but sounds reasonable. I will fix it